app/uk/gov/hmrc/helptosave/connectors/EligibilityCheckConnector.scala

```diff
@@ -65,17 +70,22 @@ class EligibilityCheckConnectorImpl @Inject() (http: WSHttp, metrics: Metrics, p
                 }, _ ⇒
                   logger.info(s"Call to check eligibility successful, received 200 (OK) ${timeString(time)}", nino)
                 )
-                result
+                Some(result)
+
+              case Status.NOT_FOUND ⇒
+                pagerDutyAlerting.alert("Retrieved nino has not been found in DES, so user is not receiving Working Tax Credit")
```

Comment:
do we really want a pager duty alert for this? It isn't an error per se is it?

Reply:
No we don't, sorry that was an overlook